packages/functional-tests/tests/settings/password.spec.ts

```diff
@@ -120,6 +120,7 @@ test.describe('password strength tests', () => {
 
     //Submit a common password
     await login.setPassword('password123123');
+    await login.submitButton.click();
```

Comment:
this is needed to get the password validation to update; it's consistent with the tests directly above in this file.